flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -288,28 +276,6 @@
 			.withDescription("Fraction of Total Flink Memory to be used as Managed Memory, if Managed Memory size is not"
 				+ " explicitly specified.");
 
-	/**
-	 * Off-Heap Managed Memory size for TaskExecutors.
-	 */
-	public static final ConfigOption<String> MANAGED_MEMORY_OFFHEAP_SIZE =
```

Comment:
other options descriptions should be also adjusted to not mention on/off-heap managed memory, just saying that it is always off-heap